setup.py

```diff
@@ -17,10 +18,71 @@
 # You should have received a copy of the GNU General Public License
 # along with ShinySDR.  If not, see <http://www.gnu.org/licenses/>.
 
-from setuptools import find_packages, setup
+import urllib
+import subprocess
 
+import setuptools.command.build_py
+from setuptools import find_packages, setup, Command
+
+ASSETS = {
+    'http://requirejs.org/docs/release/2.1.22/comments/require.js': 'shinysdr/deps/require.js',
+    'https://raw.githubusercontent.com/requirejs/text/646db27aaf2236cea92ac4107f32cbe5ae7a8d3a/text.js': 'shinysdr/deps/text.js'
+}
+
+
+class DownloadAssets(Command):
+    """Support setup.py retrieving assets from external sites"""
+
+    description = 'download assets from external sites'
+    user_options = []
+
+    @staticmethod
+    def status(s):
```

Comment:
From what I've seen, setup.py doesn't normally print things in bold. Just delete this extra code and use regular print() in both places. Keep the fluff down.